src/views/home/components/transaction-form/transaction-form.view.tsx

```diff
@@ -35,7 +42,7 @@ const TransactionForm: FC<TransactionFormProps> = ({
   const [error, setError] = useState<string>();
   const [balanceFrom, setBalanceFrom] = useState(BigNumber.from(0));
   const [balanceTo, setBalanceTo] = useState(BigNumber.from(0));
-  const [transactionData, setTransactionData] = useState<TransactionData | undefined>(transaction);
+  const [transactionData, setTransactionData] = useState<FormData | undefined>(transaction);
```

Comment:
I would rename this `[transactionData, setTransactionData]` to `[formData, setFormData]` to be consistent as well.